ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -69,6 +69,18 @@ public static void scrub(Map<String, String> attributes, ObjectNode json)
             .filter(JsonNode::isTextual) //
             .anyMatch(j -> j.textValue().startsWith("webIsolated="))) {
       handleBug("1562011", ScrubAction.DROP);
+    } else if ("c0ffeec0-ffee-c0ff-eec0-ffeec0ffeec0".equals(attributes.get(Attribute.CLIENT_ID))
```

Comment:
We should make this client id a constant, so that we can't misspell it.

Reply:
I put it into a constant and refactored it out into a separate message which makes it hopefully a little more readable